Resources/doc/index.rst

```diff
@@ -42,7 +42,7 @@ Data fixtures are PHP classes where you create objects and persist them to the
 database.
 
 Imagine that you want to add some ``Product`` objects to your database. No problem!
-Just create a fixtures class and start adding products!
+Just create a fixtures class and start adding products.
```

Comment:
I retargeted to 3.3.x, but this causes a conflict. Please remove that change.

Reply:
OK, done :-)